packages/pricing-table/src/components/SalablePricingTableReact/SalablePricingTableReact.tsx

```diff
@@ -25,7 +25,9 @@ export const SalablePricingTableReact: FC<IPricingTableReact> = ({ envConfig, ch
         if (window.SalablePricingTable) {
           (async () => {
             pricingTable = new window.SalablePricingTable(envConfig, checkoutConfig);
-            await pricingTable?.init();
+            document.querySelector('.salable-pricing-table-container')
```

Comment:
Can we put this inside if and then initialise the pricing table

Reply:
Fixed in upcoming commit